components/director/internal/persistence/const.go

```diff
@@ -5,6 +5,7 @@ import "github.com/lib/pq"
 type persistenceCtxKey string
 
 const (
+	connStringf string = "host=%s port=%s user=%s password=%s dbname=%s sslmode=%s"
```

Comment:
I don't like the location of `connStringf`, because it's related to config only. It would be more readable to keep all those things together.